apps/www/components/Climatelab/LandingPage/Page.tsx

```diff
@@ -59,18 +59,7 @@ const LandingPage = () => {
                 {t('ClimateLandingPage/content/paragraph2')}
               </p>
               <p {...styles.text}>
-                {t.elements('ClimateLandingPage/content/paragraph3', {
-                  eventsLink: (
-                    <A
-                      style={{ textDecoration: 'underline' }}
-                      href={t('ClimateLandingPage/content/eventsLinkHref')}
```

Comment:
Assuming that this is not used anywhere else `ClimateLandingPage/content/eventsLinkHref` and `ClimateLandingPage/content/eventsLinkText` should be removed from translation file as well.